#!/usr/bin/env python
PACKAGE = "apc_pcl"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_dx", double_t, 0, "min_dx", 0.1 , -2.0, 2.0)
gen.add("min_dy", double_t, 0, "min_dy", 0.02, -2.0, 2.0)
gen.add("min_dz", double_t, 0, "min_dz", 0.03, -2.0, 2.0)
gen.add("max_dx", double_t, 0, "max_dx", 0.25, -2.0, 2.0)
gen.add("max_dy", double_t, 0, "max_dy",-0.1 , -2.0, 2.0)
gen.add("max_dz", double_t, 0, "max_dz",-0.05, -2.0, 2.0)

exit(gen.generate(PACKAGE, "passthrough_filter", "Bin"))
